TimeIsMoney/TimeIsMoneyMasterViewController.h

```diff
@@ -7,23 +7,31 @@
 //
 
 #import <UIKit/UIKit.h>
+#import "AppDelegate.h"
 
-#import "TimeIsMoneySettingsModel.h"
+typedef enum timerState
+{
+    STOPPED,
+    PAUSED,
+    RUNNING_TASK,
+    RUNNING_BREAK,
+} TimerState;
 
 @interface TimeIsMoneyMasterViewController : UIViewController
 {
+    TimerState previousState;
+    TimerState timerState;
     @private
     NSTimer *countdownTimer;
     int remainingTicks;
     int pauseTime;
-    BOOL isCountdownTimerPaused;
 }
 
 @property (weak, nonatomic) IBOutlet UILabel *timeLabel;
```

Comment:
I'm not 100% sure, but : I think you can declare these iboutlet properties in the .m ... no other file should need visibility to your gui elements.